spec/jstests-spec.js

```diff
@@ -361,29 +361,53 @@ describe("Check correct url params", function () {
     expect(constructedUrl.get("mutationNum")).toBe("1");
 
     // Not on page here, should be empty
-    expect(constructedUrl.has("nodeName")).toBe(true);
-    expect(constructedUrl.get("nodeName")).toBe("");
+    expect(constructedUrl.has("nodeNames")).toBe(true);
+    expect(constructedUrl.get("nodeNames")).toEqual('[""]');
 
     expect(constructedUrl.has("tokenName")).toBe(true);
     expect(constructedUrl.get("tokenName")).toBe("");
   });
 
   it("passes correct nodeName when nodeName has a value", function () {
     nodeName.value = "A";
-    document.body.appendChild(nodeName);
+    document.body.appendChild(nodeName); 
 
-    const requestString = getUrl();
+    const requestString = getUrl(); 
+    const requestParams = requestString.substring(requestString.indexOf("?"));
+
+    const constructedUrl = new URLSearchParams(requestParams);
+    expect(constructedUrl.has("nodeNames")).toBe(true); 
+    expect(constructedUrl.get("nodeNames")).toEqual('["A"]');
+  });
+
+  it("passes correct nodeName when nodeName has a comma separated value", function () {
+    nodeName.value = "A,B";
+    document.body.appendChild(nodeName); 
+
+    const requestString = getUrl(); 
     const requestParams = requestString.substring(requestString.indexOf("?"));
 
     const constructedUrl = new URLSearchParams(requestParams);
-    expect(constructedUrl.has("nodeName")).toBe(true);
-    expect(constructedUrl.get("nodeName")).toBe("A");
+    expect(constructedUrl.has("nodeNames")).toBe(true); 
+    expect(constructedUrl.get("nodeNames")).toEqual('["A","B"]');
+  });
+
+    it("passes correct nodeName when nodeName has a comma separated value and spaces", function () {
+    nodeName.value = "  Annie boo  , B";
```

Comment:
Interesting file name choice :)

Reply:
idk why this was the first thing that came to mind!!!